src/components/Kv/KvFlag.vue

```diff
@@ -0,0 +1,1038 @@
+<template>
+	<div :class="`flag-icon ${country.toLowerCase()}`"></div>
+</template>
+
+<script>
+export default {
+	props: {
+		country: { type: String, required: true }
+	},
+};
+</script>
+
+<style lang="scss">
```

Comment:
I think the scoped attribute would be appropriate here.

Reply:
Cool, I just updated the PR. I like that better 👍